code/modules/mob/living/say.dm

```diff
@@ -14,6 +14,7 @@ var/list/department_radio_keys = list(
 	  ":u" = "Supply",		"#u" = "Supply",		".u" = "Supply",
 	  ":z" = "Service",		"#z" = "Service",		".z" = "Service",
 	  ":p" = "AI Private",	"#p" = "AI Private",	".p" = "AI Private",
+	  ":o" = "Response Team", "#o" = "Response Team", ".o" = "Response Team",

```

Comment:
I am fairly sure that 'o' does conflict with something. I can't remember what, though. I do however remember trying every letter of the alphabet, plus a bunch of symbols, in the past, and finding that essentially everything conflicts with something. 

You need to actually test/check this, and verify it doesn't conflict - not just 'hope' it doesn't.

Reply:
Well I mean, I did test it in game and it worked just fine. I am not sure how else I am supposed to test it aside from that? How did you find that every letter conflicts with something, exactly?

Reply:
https://github.com/ParadiseSS13/Paradise/blob/9c604e2c7c9e627ac34fd37aabf678346a32fd81/code/modules/mob/language.dm#L155 oops, conflicts with unathi

Reply:
Okay, time for my backup idea of using !, I guess.